packages/grafana-ui/src/components/Forms/Select/SelectOptionGroup.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { PureComponent } from 'react';
 import { css, cx } from 'emotion';
 import { GrafanaTheme } from '@grafana/data';
-import { GroupProps } from 'react-select/lib/components/Group';
+import { GroupProps } from 'react-select/src/components/Group';
```

Comment:
Not 'react-select'?

Reply:
`react-select` is used for types, `@types/react-select`

Reply:
```suggestion
import { GroupProps } from 'react-select';
```

Reply:
I tired it, it doesn't export GroupProps :/

Reply:
@tskarhed would be good to try figuring a way to use @types/react-select with Torkel's fork